src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/CommentTabPanel.kt

```diff
@@ -87,10 +83,12 @@ class CommentTabPanel(
                 .build()
                 .perform()
             driver.wait(
-                timeout = Duration.ofSeconds(60),
```

Comment:
Is there a direct need to mark timeout and condition parameters? If not I think it will be better to stick with the convention used before - just passing the parameters. One way of another, it would be the best to stick with one convention, unless for some reason it is not sensible.